tests/Checks/FileWritableCheckTest.php

```diff
@@ -15,7 +15,7 @@ class FileWritableCheckTest extends SapphireTest
 {
     public function testCheckReportsWritablePaths()
     {
-        $check = new FileWriteableCheck(TEMP_FOLDER);
+        $check = new FileWriteableCheck(sys_get_temp_dir());
```

Comment:
Fixes error `Error: Undefined constant "SilverStripe\EnvironmentCheck\Tests\Checks\TEMP_FOLDER"`

Reply:
`sys_get_temp_dir()` is not the temp dir we use for Silverstripe CMS.

Should use `TEMP_PATH`. `TEMP_FOLDER` was deprecated in 5 and removed in 6 because it was a duplicate of `TEMP_PATH`.

Reply:
Updated